cardano-node/src/Cardano/CLI/Tx.hs

```diff
@@ -144,6 +144,15 @@ withRealPBFT co cc action = do
     proto@Consensus.ProtocolRealPBFT{} -> action proto
     _ -> throwIO $ ProtocolNotSupported (coProtocol co)
 
+-- | Return an id of GenTx.
+byronGenTxId
```

Comment:
@denisshevchenko This function exists already, it is called `txId`

Reply:
I've modified the code to use `txId`.